pkg/storage/s3/s3.go

```diff
@@ -46,11 +50,23 @@ func NewDriver(c *imageregistryv1.ImageRegistryConfigStorageS3, listers *regopcl
 // getS3Service returns a client that allows us to interact
 // with the aws S3 service
 func (d *driver) getS3Service() (*s3.S3, error) {
+	ctx := context.TODO()
```

Comment:
I wasn't sure how far up I should push this context.  Can we take it all the way up to `cmd/cluster-image-registry-operator` without too much trouble?